.github/workflows/run-shared-tests.yml

```diff
@@ -75,10 +75,11 @@ jobs:
           TARGET_URL_FORM_RUNNER: ${{inputs.e2e_tests_target_url_form_runner}}
           TARGET_URL_ASSESSMENT: ${{inputs.e2e_tests_target_url_assessment}}
       - name: Upload E2E Test Report
+        if: success() || failure()
         uses: actions/upload-artifact@v3
         with:
           name: e2e-test-report
-          path: ./results
+          path: ./funding-service-design-e2e-checks/results
```

Comment:
Have we tested this is definitely the correct path?

Reply:
Ah, looks like it from the failed report, let's give this a go.

Reply:
@gidsg I'm testing from the frontend repo:
https://github.com/communitiesuk/funding-service-design-frontend/actions/runs/4183625596

Will update here if this works.

Reply:
@gidsg It worked:
https://github.com/communitiesuk/funding-service-design-frontend/actions/runs/4183956425

Will merge this PR.